### ISSUE TITLE
playerctld has no documentation

### ISSUE THREAD

**Issue by sersorrel** [altdesktop/playerctl#194]
I was running into the issue described by #128, and was about to write a script to keep track of the most-recent player, until I realised that that's playerctld. But there's no manual page for it, and no mention of it in the playerctl manual page. It would probably be more discoverable if either of those things were added.

**Comment by acrisci** [comment]
Yeah, it needs documentation. I considered it too experimental to document for awhile, but now I consider it pretty stable.
